jsondiff.go

```diff
@@ -58,17 +59,26 @@ func CompareJSON(expectedJSON []byte, actualJSON []byte, noise map[string][]stri
 // expectedJSON: The JSON string containing the expected values.
 // actualJSON: The JSON string containing the actual values.
 // Returns a Diff struct containing the colorized differences for the expected and actual JSON responses.
```

Comment:
add a comment in description as well ~ helps in auto generated documentation https://pkg.go.dev/github.com/keploy/jsonDiff